test/wargames/ropsten.test.ts

```diff
@@ -389,6 +404,82 @@ describe("ropsten deployment", function () {
     expect(changeInFlan.toString()).to.equal(impliedFlan)
   })
 
+  describe("gas test", () => {
+
+
+    this.afterEach(async () => {
+      const hre = await import("hardhat");
+      const provider = hre.network.provider;
```

Comment:
Is it necessary to import `HRE` dynamically after each test? I would assume that importing it once at the file top level and then calling `hre.network.provider.send('evm_automine', [false])` should work the same here and potentially would be more performant.

Reply:
Added a commit below to clean this up.